core/src/defaultconfigs/bareos-dir.d/fileset/LinuxAll.conf.in

```diff
@@ -24,6 +24,8 @@ FileSet {
     File = @workingdir@
     File = @archivedir@
     File = /proc
+    File = /dev
+    File = /sys
```

Comment:
What lead to this change ? Why do we want to back up device files ?

Reply:
We are in Exclude section so we explicitly show people to not try to backup those dynamically filed on run paths.
In reality, as we only use real FS Type, we could remove those but then also /proc

Reply:
I think this is fine then!